### PR TITLE
Feature/seo title

### PR DESCRIPTION
**Pull request by avi312singh**
updated some dependancies
page seo title coming in from cms 
validations done in cms

### CODE REVIEW

**Comment by avi312singh** [comment]
waiting for some further clarification from lucy and harriett

File: src/templates/blog-post.jsx [hunk 1]
```diff
@@ -231,6 +231,7 @@ BlogPostTemplate.propTypes = {
         }),
         contentfulBlogPost: PropTypes.shape({
             title: PropTypes.string,
+            pageSeoTitle: PropTypes.string.isRequired,
```
Comment (eugene-matvejev) [inline src/templates/blog-post.jsx]: what the difference from title and page seo title?

File: src/templates/page-template.jsx [hunk 2]
```diff
@@ -143,20 +145,24 @@ query componentsForHomePage($slug: String!) {
...
-        <Nav />
-        {
-            data.contentfulPageTemplate.components.map(resolve)
-        }
-        <Footer />
-    </main>
-);
+const Page = ({ data }) => {
+    const page = get(data, 'contentfulPageTemplate');
+    return (
```
Comment (eugene-matvejev) [inline src/templates/page-template.jsx]: sync with develop :) and in this case  this get is not needed

File: src/templates/blog-post.jsx (hunk 1 above)
Comment (avi312singh) [reply src/templates/blog-post.jsx]: title is the name of the content in contentful.. page seo title is the seo title applied to the page

File: src/templates/page-template.jsx (hunk 2 above)
Comment (avi312singh) [reply src/templates/page-template.jsx]: done :)

Comment (avi312singh) [reply src/templates/page-template.jsx]: it should be using destruction now

**Pull request merged**